changelogs/fragments/ansible_connection_path.yml

```diff
@@ -0,0 +1,8 @@
+bugfixes:
+  - networking connection plugins - The correct Ansible networking connection helper is now always used.
```

Comment:
```suggestion
  - persistent connection plugins - The correct Ansible persistent connection helper is now always used.
```

Reply:
Since we dumped the public script entrypoint, we needed to invent a name for it that wasn't `ansible-connection` - I like your name better than ours, so WFM. :laughing: